.travis.yml

```diff
@@ -53,24 +59,24 @@ env:
       SERVER_PROCESS=server.ts
       IMAGE=openruntimes/deno:${VERSION}-1.21
       ARCH=linux/amd64
-    - RUNTIME=deno-1.14
-      PHP_CLASS=Deno114
-      ENTRYPOINT=tests.ts
-      SERVER_PROCESS=server.ts
-      IMAGE=openruntimes/deno:${VERSION}-1.14
-      ARCH=linux/amd64
-    - RUNTIME=deno-1.13
-      PHP_CLASS=Deno113
-      ENTRYPOINT=tests.ts
-      SERVER_PROCESS=server.ts
-      IMAGE=openruntimes/deno:${VERSION}-1.13
-      ARCH=linux/amd64
-    - RUNTIME=deno-1.12
-      PHP_CLASS=Deno112
-      ENTRYPOINT=tests.ts
-      SERVER_PROCESS=server.ts
-      IMAGE=openruntimes/deno:${VERSION}-1.12
-      ARCH=linux/amd64
+    # - RUNTIME=deno-1.14
+    #   PHP_CLASS=Deno114
+    #   ENTRYPOINT=tests.ts
+    #   SERVER_PROCESS=server.ts
+    #   IMAGE=openruntimes/deno:${VERSION}-1.14
+    #   ARCH=linux/amd64
+    # - RUNTIME=deno-1.13
+    #   PHP_CLASS=Deno113
+    #   ENTRYPOINT=tests.ts
+    #   SERVER_PROCESS=server.ts
+    #   IMAGE=openruntimes/deno:${VERSION}-1.13
+    #   ARCH=linux/amd64
+    # - RUNTIME=deno-1.12
+    #   PHP_CLASS=Deno112
+    #   ENTRYPOINT=tests.ts
+    #   SERVER_PROCESS=server.ts
+    #   IMAGE=openruntimes/deno:${VERSION}-1.12
+    #   ARCH=linux/amd64
```

Comment:
I believe this needs to be uncommented ?

Reply:
I've rather added new Deno 1.24 runtime, don't think maintaining and updating all these old deno runtimes is fruitful. Deno has lot newer releases `deno 1.21` and `deno 1.24`. let me know what you think

Reply:
yes, this is a tedious process maintaining a lot of similar releases and is not worth it... 
Let's remove the old ones and keep just the latest 1 and an LTS